roles/manifest/tasks/upload_manifest.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Upload Subscription Manifest to Foreman
+  theforeman.foreman.subscription_manifest:
```

Comment:
Are we able to avoid declaring the collection explicitly here to allow re-use across downstream collections?